packages/react-components/lib/confirmation-dialog.tsx

```diff
@@ -77,7 +76,7 @@ export function ConfirmationDialog({
           <Button
             variant="contained"
             type="submit"
-            color="primary"
+            color="secondary"
```

Comment:
Any reason why this was changed to secondary?

Reply:
with the primary color it does not look like a clear CTA button when there are at least 3 buttons so I thought for the submit button to take on the secondary color so that it stands out
![image](https://user-images.githubusercontent.com/28668944/140001254-137a1fbe-4b42-4f19-a512-e92ae0c1541f.png)

and also initially the cancel button was outlined with the secondary color, which does not show up clearly for dark mode so I removed the secondary color class
![image](https://user-images.githubusercontent.com/28668944/140001745-12cea2c2-04ce-4cf4-bd06-eb068ed4460d.png)

Reply:
> with the primary color it does not look like a clear CTA button when there are at least 3 buttons so I thought for the submit button to take on the secondary color so that it stands out

That looks like a problem with the theme, if we look at the material default theme, the primary color is blue, with a white background, there is no problem of it blending into the background. The way the dark theme colors are chosen makes it have less contrast, if that is a problem then we should change the theme colors, and not the component design.

> and also initially the cancel button was outlined with the secondary color, which does not show up clearly for dark mode so I removed the secondary color class

That looks like a theme problem as well, again comparing it to the default theme, there is no readability problem. I like removing the secondary color though, it makes it blend into the background more, enforcing that the OK button should be the one the user is most interested in clicking. Also the secondary color is not meant to be a "negative" color, so using it as such is probably wrong. Still, the theme's secondary color should be fixed as it should not result in readability problems.

Reply:
ok i will revise the theme colors and update this!